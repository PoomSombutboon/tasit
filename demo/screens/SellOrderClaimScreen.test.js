import React from "react";
import NavigationTestUtils from "react-navigation/NavigationTestUtils";
import { shallow } from "enzyme";
import { SellOrderClaimScreen } from "./SellOrderClaimScreen";

describe("SellOrderClaimScreen", () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it("renders the component", async () => {
    const sellOrder = {
      id: 123,
      priceMana: 0,
      priceUSD: 0,
      asset: {
        id: -1,
        name: "Not found",
        img: "https://decentraland.org/images/logo-65f7b27caf.png",
      },
    };
    const navigation = { getParam: () => sellOrder };
    const claimSellOrder = () => {};
    expect(
      shallow(
        <SellOrderClaimScreen
          navigation={navigation}
          claimSellOrder={claimSellOrder}
        />
      )
    ).toMatchSnapshot();
  });
});
