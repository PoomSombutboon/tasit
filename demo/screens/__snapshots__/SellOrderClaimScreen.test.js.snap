// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SellOrderClaimScreen renders the component 1`] = `
<SellOrderClaim
  onClaim={[Function]}
  sellOrder={
    Object {
      "asset": Object {
        "id": -1,
        "img": "https://decentraland.org/images/logo-65f7b27caf.png",
        "name": "Not found",
      },
      "id": 123,
      "priceMana": 0,
      "priceUSD": 0,
    }
  }
/>
`;
